File: Ruby/native.rb
require 'uri'
require 'net/http'

url = URI("https://app.binds.co/api/seeds")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true
http.verify_mode = OpenSSL::SSL::VERIFY_NONE

request = Net::HTTP::Post.new(url)
request["content-type"] = 'application/json; charset=UTF-8'
request.body = "{\"collector\": \"56a351796b80339c09ff1b76\", \"from\": {\"name\": \"Charlie Parker\", \"email\": \"hello@binds.co\"}}"

response = http.request(request)
puts response.read_body

File: php/Native.php
<?php

// ask Binds.co for your own token
$token = '56a351796b80339c09ff1b76';

// preparing request body
$requestBody = array(
  'collector' => $token,
  'from' => array(
    'name' => 'Charlie Parker',
    'email' => 'hello@binds.co'
  )
);

// creating request context
// (see more at http://www.php.net/manual/en/context.http.php)
$context = stream_context_create(array(
  'http' => array(
    'method' => 'POST',
    'header' => "Content-Type: application/json;charset=UTF-8\r\n",
    'content' => json_encode($requestBody)
  )
));

// sending request
$response = file_get_contents('https://app.binds.co/api/seeds', FALSE, $context);

// checking errors
if($response === FALSE){
  die('Error');
}

// response decoding
$responseData = json_decode($response, TRUE);

// showing response data
print_r($responseData);
